ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/service/impl/ToolListCallbackConverter.java

```diff
@@ -0,0 +1,32 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.service.impl;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.web.bindery.autobean.shared.AutoBean;
+import com.google.web.bindery.autobean.shared.AutoBeanCodex;
+import org.iplantc.de.client.models.tool.ToolAutoBeanFactory;
+import org.iplantc.de.client.models.tool.Tool;
+import org.iplantc.de.client.models.tool.ToolList;
+import org.iplantc.de.client.services.converters.AsyncCallbackConverter;
+
+import java.util.List;
+
+/**
+ * Created by aramsey on 10/28/15.
+ */
+
```

Comment:
There are a number of places where the class javadoc has empty lines before the class declaration. For style consistency, the spaces should be removed.